.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish-package:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: moneymeets/action-setup-python-poetry@master
+
+      - name: Publish package
+        run: |
+          sed -i -e "s/1+SNAPSHOT/0.$(date +"%Y%m%d%H%M")+$(git rev-parse --verify --short=7 HEAD)/" pyproject.toml
```

Comment:
Do we really want to take the date the package was built, or rather the date of the HEAD commit? I think there is a subtle difference and I wonder which choice works better for us. We can merge for now as is, but maybe still something to think about later, if this wasn't already discussed.

Reply:
I like this idea! Can't find any drawbacks in it at the moment. @zyv what do you think?

Reply:
Decision: release date.